types/backbone/index.d.ts

```diff
@@ -534,15 +534,15 @@ declare namespace Backbone {
         model?: TModel;
         // TODO: quickfix, this can't be fixed easy. The collection does not need to have the same model as the parent view.
         collection?: Collection<any>; // was: Collection<TModel>;
-        el?: any;
-        events?: EventsHash;
+        el?: HTMLElement | JQuery | string;
         id?: string;
+        attributes?: { [id: string]: any };
         className?: string;
         tagName?: string;
-        attributes?: { [id: string]: any };
+        events?: EventsHash | (() => EventsHash);
```

Comment:
I'd like to see `T | (() => T)` in more places. Backbone uses this so much that it probably warrants a `Result<T>` helper type (which might already exist in `@types/underscore`, though I'm not sure).

My experience with trying to overriding such a member in a derived class hasn't been the best, though. This might work a bit better once we have these interfaces split over an interface part and a class part.